### ISSUE TITLE
Make relations test more strict - flag cases where an ontology uses RO with inconsistent semantics

### ISSUE THREAD

**Issue by cmungall** [OBOFoundry/OBO-Dashboard#136]
The BTO folks are using an RO relation "part of" without declaring it transitive:

https://github.com/BRENDA-Enzymes/BTO/issues/43

This means that anyone who is using OWL reasoning will get mysteriously incomplete examples (people who use pre OWL graph walking are immune to the problem).

There is no indication of this problem here:
<img width="1046" alt="image" src="https://github.com/user-attachments/assets/f01b2395-cf79-4e13-92c2-0931b04f58e1">

I think both BTO folks and users of BTO would welcome the ability to spot these kinds of issues ahead of time, and I am sure this applies to many ontologies too.

**Comment by cmungall** [comment]
Note: if you are coming from a formal OWL background you might interpret what I'm asking for is that all modules must be extracted using SLME. This is logical but too onerous. I think as a practical common sense requirement we can come up with a certain set of axiom types that MUST be preserved for RO

**Comment by anitacaron** [comment]
In the dashboard, we consider the base artefact, which doesn't have any axiom from RO. If it is not available, we generate one from the full.

**Comment by cmungall** [comment]
Good point. Ultimately I think we need two sets of (possibly overlapping) tests, one for base and one for release. But appreciate this might be a big change!
